src/applications/vaos/tests/services/appointment/index.v2.unit.spec.js

```diff
@@ -122,6 +122,7 @@ describe('VAOS Appointment service', () => {
             value: 'CANCELLED BY PATIENT',
           },
           { op: 'remove', path: ['practitioners'] },
+          { op: 'replace', path: ['cancellationReason'], value: 'pat' },
```

Comment:
We should probably update the mocking code in here to avoid having to add this difference. I'll make a ticket for that, it's just a minor test code issue.